pkg/auth/requests/authenticate.go

```diff
@@ -272,12 +297,34 @@ func (a *tokenAuthenticator) TokenFromRequest(req *http.Request) (*v3.Token, err
 	}
 
 	lookupUsingClient := false
+
+	if extTokenName, found := strings.CutPrefix(tokenName, "ext/"); found {
+		// Ext token detected. Perform roughly the same process as for legacy tokens, using
+		// a different store.  No indexer/cache in play here.
+
+		storedToken, err := a.extTokenStore.Get(extTokenName, "", &metav1.GetOptions{})
+		if err != nil {
+			if apierrors.IsNotFound(err) {
+				return nil, ErrMustAuthenticate
+			}
+			return nil, errors.Wrapf(ErrMustAuthenticate,
+				"failed to retrieve auth token, error: %v", err)
```

Comment:
This should be `fmt.Errorf` ?

There are a number of cases in this file (you're introducing `fmt.Errorf` later)

Reply:
Changed the ext token code to use `fmt.Errorf`. Leaving pre-existing code untouched. 
commit 7f21c334bcfc2119a87a75bbec265eacd80e165d